kymatio/scattering1d/backend/torch_backend.py

```diff
@@ -40,7 +41,7 @@ def subsample_fourier(x, k):
     res = x.view(x.shape[:-2] + (k, N // k, 2)).mean(dim=-3)
     return res
 
-def pad_1d(x, pad_left, pad_right, mode='constant', value=0.):
+def pad(x, pad_left, pad_right, mode='reflect', value=0.):
```

Comment:
Also add defaults for 'pad_left', 'pad_right'.

Reply:
Yes. Set to zero as before.